### PR TITLE
[Teams] new additions to docs

### PR DESCRIPTION
**Pull request by abracchi-tw**
- Add Teams roles to dev docs
- TEAM-565
- TEAM-409
- TEAM-571

### CODE REVIEW

File: products/cloudflare-one/src/content/cloudflare-teams-roles-permissions.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+order: 20
+---
+
+# Roles and permissions
```
Comment (abracchi-tw) [inline products/cloudflare-one/src/content/cloudflare-teams-roles-permissions.md:5]: @abelinkinbio here's the page

File: products/cloudflare-one/src/content/identity/idp-integration/google.md [hunk 2]
```diff
@@ -4,6 +4,12 @@ order: 12
 
 # Google
 
+<Aside header="Note">
```
Comment (abracchi-tw) [inline products/cloudflare-one/src/content/identity/idp-integration/google.md:7]: @kennyj42 here's the note about not needing to be a GCP user to set up GSuite/Google idps

File: products/cloudflare-one/src/content/identity/index.md [hunk 3]
```diff
@@ -6,7 +6,11 @@ order: 3
 
 Cloudflare for Teams integrates with your organization's identity provider to apply Zero Trust and Secure Web Gateway policies. If you work with partners, contractors, or other organizations, you can integrate multiple identity providers simultaneously.
 
-You can also build rules based on device posture using mutual TLS authentication, third party integrations, or the Cloudflare WARP client.
+As an alternative to configuring an identity provider, Cloudflare for Teams can send a [one-time PIN (OTP)](/identity/one-time-pin) to approved email addresses. No configuration needed — simply add a user's email address to a [Zero Trust policy](/policies/zero-trust) and to the group that allows your team to reach the application.
```
Comment (abracchi-tw) [inline products/cloudflare-one/src/content/identity/index.md:9]: @kennyj42 and here's the paragraph on OTP

File: products/cloudflare-one/src/content/cloudflare-teams-roles-permissions.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+When creating a Cloudflare for Teams account, you will be given the Super Administrator role. With this role, you can invite members to join your Teams account, and assign them different roles depending on the permissions you’d like to give them.
+
+A member can be given one of six roles within a Teams account:
+
+<TableWrap>
+
+| Role | Permissions |
+| ---- | ----------- |
+| Super Administrator - All Privileges | <ul><li>Can edit any Cloudflare setting.</li><li>Can make purchases, update billing, and manage memberships.</li><li>Can revoke the access of other Super Administrators.</li></ul> |
+| Administrator | <ul><li>Can access the full Cloudflare except for membership management and billing.</li></ul> |
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/cloudflare-teams-roles-permissions.md]: Does this lack an "account" in "Can access the full Cloudflare account"?

File: products/cloudflare-one/src/content/identity/one-time-pin.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+---
+order: 1
+---
+
+# OTP login
+
+Cloudflare Access can send a one-time PIN (OTP) to approved email addresses as an alternative to integrating an identity provider. You can simultaneously configure OTP login and the identity provider of your choice to allow users to use their own authentication method.
+
+For example, if your team uses Okta® but you’re collaborating with someone outside your organization, you can use OTP to grant  access to external resources.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/identity/one-time-pin.md]: Extra space in "OTP to grant access"

File: products/cloudflare-one/src/content/cloudflare-teams-roles-permissions.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+Only Enterprise users can assign multiple roles to additional members.
+
+</Aside>
+
+To check the list of members in your account, or to add new members:
+
+1. Navigate to the Cloudflare Dashboard.
+1. From your Account Home, navigate to Members.
+1. Enter a member’s email address to add them to your account, and click Invite.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/cloudflare-teams-roles-permissions.md]: I feel "Invite" should be bold

File: products/cloudflare-one/src/content/identity/one-time-pin.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+## Set up OTP
+
+To set up OTP to allow guest-user access, you simply add the guest user's email address to a [Zero Trust policy](/policies/zero-trust/policy-management#add-a-policy) and to the [group](/identity/users/groups) that allows your team to reach the application.
+
+This is the workflow for your guest user:
+
+1. When logging in to your application, they will select OTP.
+2. Then, they will enter their email address and click **Send me a code**.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/identity/one-time-pin.md]: This steps and the ones below should be changed to 1. ?

File: products/cloudflare-one/src/content/tutorials/rdp.md [hunk 8]
```diff
@@ -76,21 +76,23 @@ Ensure that the machine's firewall permits egress on ports `80`, `443`, and `338
...
 
-Run the following command to authenticate cloudflared into your Cloudflare account.
+1. Run the following command to authenticate cloudflared into your Cloudflare account.
 
-```sh
-C:\Cloudflared\bin\cloudflared.exe login
-```
+ ```txt
+ C:\Cloudflared\bin\cloudflared.exe login
+ ```
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/tutorials/rdp.md:83]: Should this be indented to not break the automatic numbering in this list?

File: products/cloudflare-one/src/content/tutorials/rdp.md [hunk 9]
```diff
@@ -104,35 +106,35 @@ Replacing `<NAME>` with a name for the Tunnel. This name can be any value. A sin
...
 
-Run `Notepad.exe` as an administrator. Next, configure the Tunnel, replacing the example ID below with the ID of the Tunnel created above. Additionally, replace the hostname in this example with the hostname of the application configured with Cloudflare Access.
+1. Run `Notepad.exe` as an administrator. Next, configure the Tunnel, replacing the example ID below with the ID of the Tunnel created above. Additionally, replace the hostname in this example with the hostname of the application configured with Cloudflare Access.
 
-```yaml
-tunnel: 6ff42ae2-765d-4adf-8112-31c55c1551ef
-credentials-file: C:\Users\%USERNAME%\.cloudflared\6ff42ae2-765d-4adf-8112-31c55c1551ef.json
+  ```yaml
+  tunnel: 6ff42ae2-765d-4adf-8112-31c55c1551ef
+  credentials-file: C:\Users\%USERNAME%\.cloudflared\6ff42ae2-765d-4adf-8112-31c55c1551ef.json
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/tutorials/rdp.md:123]: Close with ```

File: products/cloudflare-one/src/content/tutorials/rdp.md [hunk 10]
```diff
@@ -104,35 +106,35 @@ Replacing `<NAME>` with a name for the Tunnel. This name can be any value. A sin
...
-  # Catch-all rule, which responds with 404 if traffic doesn't match any of
-  # the earlier rules
-```
+  ingress:
+    - hostname: azure.widgetcorp.tech
+      service: rdp://localhost:3389
+    - service: http_status:404
+    # Catch-all rule, which responds with 404 if traffic doesn't match any of
+    # the earlier rules
+  ```
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/tutorials/rdp.md:131]: Indent to not break automatic numbering in this list

**Review by marciocloudflare — APPROVED** [review]

**Review by TownLake — APPROVED** [review]

**Pull request merged**